src/tokens/Achievements/Achievements.cairo

```diff
@@ -0,0 +1,286 @@
+    }(account: felt, id: Uint256) -> (balance: Uint256):
+    return ERC1155.balance_of(account, id)
+end
+
+@view
+func balanceOfBatch{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(
+        accounts_len: felt,
+        accounts: felt*,
+        ids_len: felt,
+        ids: Uint256*
+    ) -> (balances_len: felt, balances: Uint256*):
+    let (balances_len, balances) =  ERC1155.balance_of_batch(accounts_len, accounts, ids_len, ids)
+    return (balances_len, balances)
+end
+
+@view
+func isApprovedForAll{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(account: felt, operator: felt) -> (isApproved: felt):
+    let (is_approved) = ERC1155.is_approved_for_all(account, operator)
+    return (is_approved)
+end
+
+@view
+func owner{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }() -> (owner: felt):
+    let (owner: felt) = Ownable.owner()
+    return (owner)
+end
+
+@view
+func paused{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }() -> (paused: felt):
+    let (paused) = Pausable.is_paused()
+    return (paused)
+end
+
+#
+# Externals
+#
+
+@external
+func setURI{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(uri: felt):
+    Ownable.assert_only_owner()
+    ERC1155._set_uri(uri)
+    return ()
+end
+
+@external
+func setApprovalForAll{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(operator: felt, approved: felt):
+    Pausable.assert_not_paused()
+    ERC1155.set_approval_for_all(operator, approved)
+    return ()
+end
+
+@external
+func safeTransferFrom{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(
+        from_: felt,
+        to: felt,
+        id: Uint256,
+        amount: Uint256,
+        data_len: felt,
+        data: felt*
+    ):
+    Pausable.assert_not_paused()
+    ERC1155.safe_transfer_from(from_, to, id, amount, data_len, data)
+    return ()
+end
+
+
+@external
+func safeBatchTransferFrom{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(
+        from_: felt,
+        to: felt,
+        ids_len: felt,
+        ids: Uint256*,
+        amounts_len: felt,
+        amounts: Uint256*,
+        data_len: felt,
+        data: felt*
+    ):
+    Pausable.assert_not_paused()
+    ERC1155.safe_batch_transfer_from(
+        from_, to, ids_len, ids, amounts_len, amounts, data_len, data)
+    return ()
+end
+
+@external
+func mint{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr
+    }(
+        to: felt,
+        id: Uint256,
+        amount: Uint256,
+        data_len: felt,
+        data: felt*
+    ):
+    Ownable.assert_only_owner()
+    let (caller) = get_caller_address()
```

Comment:
Is this necessary?

Reply:
Nope doesn't look like it. Removed both.